autoscale/src/main/java/com/sequenceiq/periscope/monitor/FlowMonitor.java

```diff
@@ -0,0 +1,38 @@
+package com.sequenceiq.periscope.monitor;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.api.endpoint.v4.common.StackType;
+import com.sequenceiq.periscope.domain.Cluster;
+import com.sequenceiq.periscope.monitor.evaluator.EvaluatorExecutor;
+import com.sequenceiq.periscope.monitor.evaluator.flow.FlowEvaluator;
+
+@Component
+@ConditionalOnProperty(prefix = "periscope.enabledAutoscaleMonitors.flow-monitor", name = "enabled", havingValue = "true")
+public class FlowMonitor extends ClusterMonitor {
+
+    @Override
+    public String getIdentifier() {
+        return "flow-monitor";
+    }
+
+    @Override
+    public String getTriggerExpression() {
```

Comment:
Every 10 minutes is OK for cleanup. However, that is likely not good enough to go and check the status of pending triggers.

Reply:
Separated out cleanup from updating status of pending triggers. Now, cleanup runs every 10 minutes -> (#13906)